ui/admin/app/components/form/credential/ssh_private_key/index.hbs

```diff
@@ -112,30 +122,36 @@
       {{#if @model.errors.private_key}}
         <F.Error as |E|>
           {{#each @model.errors.private_key as |error|}}
-            <E.Message>{{error.message}}</E.Message>
+            <E.Message data-test-error-message-private-key>
```

Comment:
None blocking comment: Not sure we should use a `data-test-` for each error message. Perhaps there is a less repetitive way of doing it? Can you check with @cameronperera what would be a good pattern here?

Not saying this is not a good one, just wondering if we could bring more eyes to this and gather opinions.